proc_echo_mw.py

```diff
@@ -27,16 +28,13 @@
     s.ift('t2') # inverse fourier transform into time domain
     logger.debug(strm("THIS IS THE SHAPE"))
     logger.debug(strm(ndshape(s)))
-    s = slice_FID_from_echo(s)['t2':(None,0.05)]
-    # visualize time domain after filtering and phasing 
-    fl.side_by_side('time domain (after filtering and phasing)\n$\\rightarrow$ use to adjust time range', s, time_range)
-    s =s['t2':time_range] # slices out time range along t2 axis
+    s = slice_FID_from_echo(s,max_t=max_t,fl=fl)    # visualize time domain after filtering and phasing
     #{{{apodizing and zero fill
     fl.next('apodize and zero fill')
-    R = 5.0/(time_range[-1]) # assume full decay by end time
+    R = 5.0/(max_t) # assume full decay by end time
```

Comment:
both instances of max_t would just be time_range[-1], following on comment above.

Reply:
again, this should show as outdated